scripts/run_nosetests_for_jenkins.sh

```diff
@@ -96,8 +96,15 @@ else
         COUNT=0
         OLD_IFS=$IFS
         IFS=$'\n'
+
+        if [ $(uname) == "Darwin" ]; then
+            LINES=$(find ${NOSE_TESTS_TO_RUN} -type f -iname "test_*.py" | gsort -R --random-source=/dev/zero | xargs -n${BUCKET_SIZE})
```

Comment:
I would rather do an `SORT=gsort -R` or otherwise on the if and then that variable it in the pipe to avoid duplicating the whole line for that minor difference.